src/core/components/Rating.vue

```diff
@@ -1,22 +1,33 @@
 <template>
-  <div class="row items-start">
+  <div class="row justify-center">
     <div class="caption input-caption">
       <p :class="[{ required: requiredField }]">{{ caption }}</p>
-      <span class="q-mx-sm">(1 : {{ firstLabel }} - 5 : {{ lastLabel }})</span>
     </div>
     <q-icon v-if="error" name="error_outline" color="secondary" class="col-1" />
   </div>
-  <q-field borderless :error="error" :error-message="errorMessage">
+  <div borderless :error="error" :error-message="errorMessage" class="container">
     <q-rating :model-value="modelValue" @update:model-value="update" :icon="icon" max="5" color="primary"
-      size="lg" class="q-mx-sm">
+      size="lg" class="q-ma-md">
       <template #tip-1>
```

Comment:
Ici j'aurais voulu faire une boucle mais je n'ai pas reussi a faire fonctionner les tootltip, j'ai l'impression que c'est pas possible de passer un v-slot qui soit pas constant

Reply:
tu peux faire un truc du style 
````
      <template v-for="(slotName, index) in ['tip-1', 'tip-2', 'tip-3', 'tip-4','tip-5' ]" #[slotName] :key="slotName">
          <q-tooltip v-if="labels[index +1]">{{ labels[index +1] }}</q-tooltip>
      </template>
````
c'est pas très beau mais j'ai pas réussi à faire avec un  v-for="[value, key] in Object.entries(labels)" parce que le nom du slot ça doit être tip-{key} et qu'on peut pas faire #tip-[key]

Reply:
Ah oui bien vu ! J'avais pas pensé boucler sur les tip, ca me semble mieux de faire ca !

Reply:
tu peux créer une variable `formattedToolTips = Object.fromEntries(Object.entries(props.labels).map(([key, value]) => [`tip-${key}`, value]));` et après tu remplaces par 
````
<template v-for="[key, value] in Object.entries(formattedToolTips)" #[key] :key="key">
  <q-tooltip v-if="value">{{ value }}</q-tooltip>
</template>
````